utils/clflags.ml

```diff
@@ -358,3 +358,18 @@ let parse_color_setting = function
 let color = ref Misc.Color.Auto ;; (* -color *)
 
 let unboxed_types = ref false
+
+let arg_spec = ref []
+let arg_names = ref Misc.StringMap.empty
+let add_arguments loc args =
+  List.iter (function (arg_name, _, _) as arg ->
+    try
+      let loc2 = Misc.StringMap.find arg_name !arg_names in
+      Printf.eprintf
+        "Warning: plugin argument %s is already defined:\n" arg_name;
```

Comment:
I guess this can be extracted from the callstack at registration time.  That said, it might be more user friendly to have each plugin provide a symbolic name and use that name in the error message.